src/interface/meshblock_data.hpp

```diff
@@ -209,6 +211,17 @@ class MeshBlockData {
     return GetAllVariables(sparse_ids).unique_ids();
   }
 
+  DataCollection<SwarmContainer> swarm_data;
```

Comment:
I don't know that this needs to be a `DataCollection<SC>` rather than just a `SwarmContainer` because I don't think we need "stages" inside an already-staged `MeshBlockData`.

Reply:
Yeah this first pass just ensured that what used to belong to `pmb` is now owned by `MeshblockData`, but as you allude to, there may be some inconsistencies that need to be resolved (see above comment).